generator/generator_config.textproto

```diff
@@ -63,6 +63,21 @@ service {
   retryable_status_codes: ["kDeadlineExceeded", "kUnavailable"]
 }
 
+# Auto ML
+service {
+  service_proto_path: "google/cloud/automl/v1/service.proto"
+  product_path: "google/cloud/automl"
+  initial_copyright_year: "2022"
+  retryable_status_codes: ["kDeadlineExceeded", "kUnavailable"]
+}
+
+service {
+  service_proto_path: "google/cloud/automl/v1/prediction_service.proto"
+  product_path: "google/cloud/automl"
+  initial_copyright_year: "2022"
+  retryable_status_codes: ["kDeadlineExceeded", "kUnavailable"]
```

Comment:
I think I would do just `kUnavailable` for prediction. 

https://github.com/googleapis/googleapis/blob/107a322d4c61b87221f89f627c73029330e17a5e/google/cloud/automl/v1/automl_grpc_service_config.json#L95-L109